pkgs/tools/security/pass/default.nix

```
@@ -111,6 +111,10 @@ stdenv.mkDerivation rec {
     # Ensure all dependencies are in PATH
     wrapProgram $out/bin/pass \
       --prefix PATH : "${wrapperPath}"
+  '' + lib.optionalString (ydotool == pkgs.wtype) ''
```

```suggestion
  '' + lib.optionalString withYdotool ''
```
We should not compare packages here and rather use an input boolean variable.